src/Umbraco.Infrastructure/Migrations/Upgrade/V_13_0_0/AddWebhookDatabaseLock.cs

```diff
@@ -24,7 +24,7 @@ protected override void Migrate()
 
         if (webhookRequestLock is null)
```

Comment:
variable should be named WebhookLogsLock as that is what is being fetched right?

Reply:
Yep 👍